package.json

```diff
@@ -19,7 +20,7 @@
     "debug": "^2.2.0",
     "lodash": "^3.9.3",
     "mpns": "^2.1.0",
-    "node-cache": "^2.2.0",
+    "node-cache": "^2.1.1",
```

Comment:
Should have commented that better, see the [release history](https://www.npmjs.com/package/node-cache#release-history) for node-cache.  The 2.2.0 version was revoked and is no longer available.  I did a quick pass over the 2.1.1 for compat but I should have spent more time reviewing it.